alert-issuetracker/build.gradle

```diff
@@ -0,0 +1,6 @@
+project.ext.moduleName = 'com.synopsys.integration.alert.issuetracker'
+
+dependencies {
+    implementation project(':alert-common')
```

Comment:
What are the main dependencies that you couldn't replace?

Reply:
SetMap is the last item.  I am cleaning that up today.